graylog2-web-interface/src/components/enterprise/EnterpriseFreeLicenseForm.jsx

```diff
@@ -104,6 +139,15 @@ export default class EnterpriseFreeLicenseForm extends React.Component {
                    placeholder="Please provide a valid email address to send the license key to"
                    required
                    onChange={this.handleInput('email')} />
+            <Input type="text"
```

Comment:
How do we feel about using [`tel`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel)?